modules/gapi/include/opencv2/gapi/gproto.hpp

```diff
@@ -126,6 +126,10 @@ bool GAPI_EXPORTS can_describe(const GMetaArgs& metas, const GRunArgs& args);
 // coincides with output arguments passed to computation in cpu and ocl backends
 bool GAPI_EXPORTS can_describe(const GMetaArg&  meta,  const GRunArgP& argp);
 
+// Validates input arguments
+bool GAPI_EXPORTS validate_input_arg(const GRunArg& arg);
+bool GAPI_EXPORTS validate_input_arg(const GRunArgs& args);
```

Comment:
for a `GRunArgs` version I'd name it `.._args`

Reply:
Fixed